docs/shared/migration/migration-angular.md

```diff
@@ -353,7 +45,7 @@ width="100%" /%}
 
 As of Nx Console version 17.15.0, Angular CLI users will receive a notice periodically when running commands via Nx Console, asking if they want to use Nx to make their Angular commands faster.
 
-When you click this button, we’ll run the [make-angular-cli-faster script](https://github.com/nrwl/nx/tree/master/packages/make-angular-cli-faster) to decorate your Angular workspace with Nx, allowing for cached builds, and for you to share this cache with your teammates via Nx Cloud.
+When you click this button, we’ll run the `nx init` command to decorate your Angular workspace with Nx, allowing for cached builds, and for you to share this cache with your teammates via Nx Cloud.
```

Comment:
yeah I was about to chime in here as I'm reviewing this. 

We don't decorate any more, but rather they get a notification about migrating (with the new message) since we're going to deprecate Nx Console for Angular CLI based projects to make maintenance easier.

Tbh I'd remove this section, or at least remove the video + include a screenshot of the new popup they might get and what happens in that case.